modules/service-fips.sh

```diff
@@ -50,6 +54,62 @@ fips_disable() {
     not_supported 'Disabling FIPS'
 }
 
+fips_update() {
+    local token="$1"
+    local bypass_prompt="$2"
+    local fips_configured=0
+
+    check_token "$FIPS_UPDATES_REPO_URL" "$token"
+
+    echo -n "Updating FIPS packages will take the system out of FIPS compliance."
+    if [ "$bypass_prompt" -ne 1 ]; then
+        if ! prompt_user 'Do you want to proceed?'; then
+            error_msg "Aborting updating FIPS packages..."
+            error_exit no_updates_selected
+        fi
+    fi
+
+    #add the fips-updates repo if the system is undergoing updates the first time
```

Comment:
please add a blank space after `#` (same for other cases below)

Reply:
Done.